src/OrchardCore.Modules/OrchardCore.Features/Controllers/AdminController.cs

```diff
@@ -63,6 +67,7 @@ public async Task<ActionResult> Features(string tenant)
                 // if the user provide an invalid tenant value, we'll set it to null so it's not available on the next request
                 tenant = settings?.Name;
                 viewModel.Name = settings?.Name;
+                viewModel.IsDefaultTenant = _shellSettings.IsDefaultShell() && (settings == null || settings.IsDefaultShell());
```

Comment:
Would be better to init a `var shellSettings = _shellSettings;` before and use it in the delegate of the child scope.

But even if `_shellSettings` is a tenant singleton here we may be in a different tenant container, so better to not capture `_shellSettings` belonging to another container. So maybe just init before `var name = _shellSettings.Name` or even more `var isDefaultShell = _shellSettings.IsDefaultShell()`, and then use it in the delegate.

Reply:
@jtkech done